### ISSUE TITLE
gtk_text_iter_get_attributes not implemented

### ISSUE THREAD

**Issue by stacksmith** [crategus/cl-cffi-gtk#51]
Would it be possible to add that?  I think the underlying structures are actually implemented already...

**Comment by Ferada** [comment]
Sorry but isn't that defined in `gtk.text-iter.lisp`? Or did you mean a different function?

**Comment by stacksmith** [comment]
@Ferada, you are right - I meant **gtk_text_iter_get_default_attributes**!  Without it, gtk-text-iter-get-attributes is not useful, as I understand the situation.

**Comment by stacksmith** [comment]
Thanks! I tried it myself, but got lost in cffi crap.  Your patch should help.
